src/test/java/jalau/at18/katas/highestlowest/sarai/MainTest.java

```diff
@@ -0,0 +1,11 @@
+package jalau.at18.katas.highestlowest.sarai;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+    public void shouldReturnHighestlowestNumbers() {
+        String input = "1 2 3 4 5";
+        String expected = "numbers 5 1";
```

Comment:
expected does not match kata requirement
```
highAndLow("1 2 3 4 5");  // return "5 1"
```
"numbers" should not be part of the result